QUEST_LV_0200.tsv

```diff
@@ -1101,11 +1101,11 @@ QUEST_LV_0200_20150317_001100	I'd be really rich if things go well this time. He
 QUEST_LV_0200_20150317_001101	{memo X}That's a lot. {nl}It sure is good enough.
 QUEST_LV_0200_20150317_001102	Now, please put up this scarecrow with this charm in my farm. {nl}At times like this, I want to watch the fun all by myself.
 QUEST_LV_0200_20150317_001103	The farm is located on the left of Arkllui Crossroads. {nl}It's a bit far but that's how stay low from the villagers.
-QUEST_LV_0200_20150317_001104	Bankrupted Bee Farmer Rosen
-QUEST_LV_0200_20150317_001105	It's all gone. I lost everything. {nl}I took care of the beehives for years and the monsters destroyed everything. {nl}The priest told me to hold it but..
-QUEST_LV_0200_20150317_001106	Are you trying to comfort me? {nl}Then nothing would be better than showing those monsters a lesson.
-QUEST_LV_0200_20150317_001107	I know the priest is right but I can still rant about it, right?
-QUEST_LV_0200_20150317_001108	That feels better. {nl}How I wish they'd be gone forever.
+QUEST_LV_0200_20150317_001104	Ruined Bee Farmer Rosen
+QUEST_LV_0200_20150317_001105	$It's all gone.. I have lost everything.. {nl}I took care of the beehives for years.. and the monsters destroyed everything. {nl}The priest told me to persevere, but..
```

Comment:
will change for this line but many others will require the same treatment sometime in the future